packages/x-date-pickers/src/locales/plPL.ts

```diff
@@ -0,0 +1,52 @@
+import { PickersLocaleText } from './utils/pickersLocaleTextApi';
+import { getPickersLocalization } from './utils/getPickersLocalization';
+import { CalendarPickerView } from '../internals/models';
+
+const plPLPickers: PickersLocaleText<any> = {
```

Comment:
It should be partial because when we modify the default translation, `PickersLocaleText<any>` type will change but we will not be able to maintain other locales by ourself

```suggestion
const plPLPickers: Partial<PickersLocaleText<any>> = {
```